proposals/source-install.md

```diff
@@ -0,0 +1,1133 @@
+   on the system. For details on the install mechanics, see the
+   [Build/install mechanics section](#buildinstall-mechanics).
+
+After the above commands complete, the AWS CLI v2 is installed at the default
+location of `/usr/local/lib/aws-cli` and creates symlinks for the `aws` and
+`aws_completer` executables in the `/usr/local/bin` directory. These
+locations  are configurable through Autotools. See the
+[Install location configuration section](#install-location) for more
+information.
+
+#### Requirements
+
+These are the requirements to build the AWS CLI v2 from source:
+
+* An environment that can run Autotools generated files (e.g.,
+  `configure`, `Makefile`). These files are widely portable across POSIX
+  platforms, and for systems that do not come with POSIX-compliant shells like
+  Windows, there is additional software available (e.g., MSYS2) that allow users
+  to run Autotools generated files. For more information, see the
+  [Appendix on Windows usage](#windows).
+
+* Python 3.8+ interpreter
+
+* (Optional) All build and runtime Python library dependencies
+  of the AWS CLI v2. This can be opted out of through configuration that
+  downloads and uses the dependencies as part of the build step.
+  See the [Downloading dependencies configuration section](#downloading-dependencies)
+  for more information.
+
+#### Configuration
+
+Configuration for the build and install of the AWS CLI v2 is specified using
+the `configure` script. For the documentation of all configuration options, run
+the `configure` script with the `--help` option:
+```bash
+./configure --help
+```
+This is a sample output from the help page:
+```
+Configures builds and installs of the AWS CLI
+
+Usage: ./configure [OPTION]... [ENV_VAR=VALUE]...
+
+Help options:
+  -h, --help              Display help
+  -V, --version           Display version
+
+Installation directories:
+  --prefix=PREFIX         Set installation prefix. By default, this value is
+                          "/usr/local".
+  --libdir=LIBDIR         Set parent directory for AWS CLI installation. The
+                          full path to the AWS CLI installation is "LIBDIR/aws-cli".
+                          The default value for "LIBDIR" is "PREFIX/lib"
+                          (i.e., "/usr/local/lib" if "--prefix" is not set).
+  --bindir=BINDIR         Set install directory for AWS CLI executables. The
+                          default value for "BINDIR" is "PREFIX/bin"
+                          (i.e., "/usr/local/bin" if "--prefix" is not set).
+
+Optional Packages:
+  --with-PACKAGE[=ARG]    use PACKAGE [ARG=yes]
+  --without-PACKAGE       do not use PACKAGE (same as --with-PACKAGE=no)
+  --with-install-type=[portable-exe|system-sandbox]
+                          Specify type of AWS CLI installation. Options are:
+                          "portable-exe", "system-sandbox" (default is
+                          "frozen-exe")
```

Comment:
If the options are `portable-exe` or `system-sandbox`, what is the default of `frozen-exe`?

Reply:
Oh that is a typo in the POC. The default should be `portable-exe`. The `frozen-exe` option does not exist and I believe that was just the name that I originally gave `portable-exe`.